_data-prepper/pipelines/configuration/processors/parse-json-processor.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Parse JSON
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Parse JSON processor
+
+The `Parse JSON` processor takes in an event and parses its JSON data, including any nested fields.
+
+## Basic usage
+
+To get started, create the following `pipelines.yaml` file:
```

Comment:
@oeyh Can we define these parameters and what we should enter?

Reply:
Which parameters are you referring to?

Reply:
@oeyh This comment has been addressed by other comments (re: pipeline.yaml file location). Can you confirm the file is `pipeline.yaml` and not `pipelines.yaml`?

Reply:
Either is OK. The name of the pipeline yaml file name is not fixed. Users can alter this file name as needed. For documentation purpose, feel free to choose one that's consistent with other places.